assets/src/common/components/featured-image-cropper.js

```diff
@@ -5,14 +5,13 @@ const { wp } = window;
  * A state for cropping a featured image.
  *
  * @constructs FeaturedImageCropper
- * @inheritDoc
  */
 const FeaturedImageCropper = wp.media.controller.Cropper.extend( {
 	/**
 	 * Creates an object with the image attachment and crop properties.
 	 *
 	 * @param {wp.media.model.Attachment} attachment The attachment to crop.
-	 * @return {jQuery.promise} A jQuery promise that represents the crop image request.
+	 * @return {Function} A jQuery promise that represents the crop image request.
```

Comment:
Shouldn't this be a `Promise`? Or is a jquery Promise too different from a _real_ one to warrant that type?

Reply:
It's not a real Promise, but an object with a similar API. Kinda like a poor man's implementation of a Promise.

According to https://api.jquery.com/Types/#Promise it doesn't have a `catch` method for example.

But since we don't really rely on the return value in our code, I think we can just use `Promise` with some updated comment to clarify it's a jQuery Promise